pkg/kubectl/cmd/create_role.go

```diff
@@ -53,6 +53,9 @@ var (
 
 	// Valid resource verb list for validation.
 	validResourceVerbs = []string{"*", "get", "delete", "list", "create", "update", "patch", "watch", "proxy", "redirect", "deletecollection", "use", "bind", "impersonate"}
+
+	// Additional resource and apiGroup
+	additionalResources = map[string]string{"users":"", "groups":"", "userextras":"authentication.k8s.io"}
```

Comment:
Seems like we'll want `map[string]GroupResource` so that we use the "normal" type for handling groupResource specification and can scope special verb/resource combinations.